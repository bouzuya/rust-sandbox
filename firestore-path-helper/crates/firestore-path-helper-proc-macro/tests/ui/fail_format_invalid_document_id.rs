fn main() {
    firestore_path_helper_proc_macro::firestore_path_helper!("users/{u$er_id}", user_id = String);
}
