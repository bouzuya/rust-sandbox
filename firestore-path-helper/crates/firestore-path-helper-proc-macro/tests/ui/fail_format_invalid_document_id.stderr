error: format contains invalid document id segment: '{u$er_id}'
 --> tests/ui/fail_format_invalid_document_id.rs:2:62
  |
2 |     firestore_path_helper_proc_macro::firestore_path_helper!("users/{u$er_id}", user_id = String);
  |                                                              ^^^^^^^^^^^^^^^^^
